week2/homework/src/index.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const program = require('commander');
+
+program
+  .version('0.0.1')
+  .command('list', 'list all the items', {file: 'list'})
+  .command('add <todo>', 'add item to the list', {file: 'add'})
+  .command('remove <index>', 'remove an item from the list by index', {file: 'remove'})
+  .command('reset', 'remove all items from list', {file: 'reset'})
+  .command('update <index> <todo>', 'update to items', {file: 'update'})
+  .parse(process.argv);
```

Comment:
👍 nice  that you are using commander. I have not seen this option to use a file before, but looks really interesting